server/src/instant/fixtures.clj

```diff
@@ -51,10 +64,21 @@
             r (resolvers/make-zeneca-resolver id)]
         (f app r)))))
 
+(defn with-zeneca-app-no-indexing [f]
```

Comment:
I use this for the indexing test--turns out the test wasn't really testing what I wanted it to because everything was already indexed.

Reply:
🫡